### ISSUE TITLE
'symbol(s) not found for architecture armv7' when trying to archive using Github Actions  

### ISSUE THREAD

**Issue by isis-hanglyde** [fastlane/fastlane#18484]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)
Saw only the following issue: 
https://github.com/fastlane/fastlane/issues/17031
##### Complete output when running fastlane, including the stack trace and command used
<!-- 
Current runner version: '2.277.1'
Operating System
Virtual Environment
GITHUB_TOKEN Permissions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v1'
Download action repository 'subosito/flutter-action@v1'
Download action repository 'maierj/fastlane-action@v2.0.1'
7s
1m 28s
46s
5m 34s
Run flutter build ios --release --no-codesign
  flutter build ios --release --no-codesign
  shell: /bin/bash -e {0}
  env:
    FLUTTER_HOME: /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64
Warning: Building for device with codesigning disabled. You will have to manually codesign before deploying to device.
Building com.hanglyde.Hanglyde for device (ios-release)...
Running pod install...                                             89.4s
Running Xcode build...                                          
Xcode build done.                                           220.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/photo_manager-0.5.8/ios/Classes/core/PMManager.m:725:42: warning: 'openURL:options:completionHandler:' is only available on iOS 10.0 or newer [-Wunguarded-availability]
          [[UIApplication sharedApplication] openURL:url
                                             ^~~~~~~~~~~
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/photo_manager/photo_manager-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:130:1: note: 'openURL:options:completionHandler:' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 9.0.0
    - (void)openURL:(NSURL*)url options:(NSDictionary<UIApplicationOpenExternalURLOptionsKey, id> *)options completionHandler:(void (^ __nullable)(BOOL success))completion API_AVAILABLE(ios(10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/photo_manager-0.5.8/ios/Classes/core/PMManager.m:725:42: note: enclose 'openURL:options:completionHandler:' in an @available check to silence this warning
          [[UIApplication sharedApplication] openURL:url
                                             ^~~~~~~~~~~
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/photo_manager-0.5.8/ios/Classes/core/PMManager.m:764:120: warning: values of type 'NSUInteger' should not be used as format arguments; add an explicit cast to 'unsigned long' instead [-Wformat]
      [PMLogUtils.sharedInstance info:[NSString stringWithFormat:@"save image with data, length: %ld, title:%@, desc: %@", data.length, title, desc]];
                                                                                          ~~~                       ^~~~~~~~~~~
                                                                                          %lu                       (unsigned long)
    2 warnings generated.
    1 warning generated.
    error: the following command failed with exit code 0 but produced no further output
    CompileC /Users/runner/Library/Developer/Xcode/DerivedData/Runner-hercuopqksatovbproiajxdfceqh/Build/Intermediates.noindex/Pods.build/Release-iphoneos/permission_handler.build/Objects-normal/arm64/PhonePermissionStrategy.o /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/PhonePermissionStrategy.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/PermissionManager.m:75:59: warning: 'openURL:' is deprecated: first deprecated in iOS 10.0 [-Wdeprecated-declarations]
            BOOL success = [[UIApplication sharedApplication] openURL:[NSURL URLWithString:UIApplicationOpenSettingsURLString]];
                                                              ^~~~~~~
                                                              openURL:options:completionHandler:
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:123:1: note: 'openURL:' has been explicitly marked deprecated here
    - (BOOL)openURL:(NSURL*)url API_DEPRECATED_WITH_REPLACEMENT("openURL:options:completionHandler:", ios(2.0, 10.0)) NS_EXTENSION_UNAVAILABLE_IOS("");
    ^
    1 warning generated.
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:48:7: warning: 'UIUserNotificationType' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
          UIUserNotificationType notificationTypes = 0;
          ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:49:28: warning: 'UIUserNotificationTypeSound' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
          notificationTypes |= UIUserNotificationTypeSound;
                               ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:50:28: warning: 'UIUserNotificationTypeAlert' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
          notificationTypes |= UIUserNotificationTypeAlert;
                               ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:51:28: warning: 'UIUserNotificationTypeBadge' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
          notificationTypes |= UIUserNotificationTypeBadge;
                               ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:52:7: warning: 'UIUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNNotificationSettings [-Wdeprecated-declarations]
          UIUserNotificationSettings *settings = [UIUserNotificationSettings settingsForTypes:notificationTypes categories:nil];
          ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:44:12: note: 'UIUserNotificationSettings' has been explicitly marked deprecated here
    @interface UIUserNotificationSettings : NSObject
               ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:52:74: warning: 'UIUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNNotificationSettings [-Wdeprecated-declarations]
          UIUserNotificationSettings *settings = [UIUserNotificationSettings settingsForTypes:notificationTypes categories:nil];
                                                                             ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:44:12: note: 'UIUserNotificationSettings' has been explicitly marked deprecated here
    @interface UIUserNotificationSettings : NSObject
               ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:53:42: warning: 'registerUserNotificationSettings:' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's -[UNUserNotificationCenter requestAuthorizationWithOptions:completionHandler:] and -[UNUserNotificationCenter setNotificationCategories:] [-Wdeprecated-declarations]
          [[UIApplication sharedApplication] registerUserNotificationSettings:settings];
                                             ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:250:1: note: 'registerUserNotificationSettings:' has been explicitly marked deprecated here
    - (void)registerUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings API_DEPRECATED("Use UserNotifications Framework's -[UNUserNotificationCenter requestAuthorizationWithOptions:completionHandler:] and -[UNUserNotificationCenter setNotificationCategories:]", ios(8.0, 10.0)) API_UNAVAILABLE(tvos);
    ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:75:5: warning: 'UIUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNNotificationSettings [-Wdeprecated-declarations]
        UIUserNotificationSettings * setting = [[UIApplication sharedApplication] currentUserNotificationSettings];
        ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:44:12: note: 'UIUserNotificationSettings' has been explicitly marked deprecated here
    @interface UIUserNotificationSettings : NSObject
               ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:75:79: warning: 'currentUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's -[UNUserNotificationCenter getNotificationSettingsWithCompletionHandler:] and -[UNUserNotificationCenter getNotificationCategoriesWithCompletionHandler:] [-Wdeprecated-declarations]
        UIUserNotificationSettings * setting = [[UIApplication sharedApplication] currentUserNotificationSettings];
                                                                                  ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:253:70: note: property 'currentUserNotificationSettings' is declared deprecated here
    @property(nonatomic, readonly, nullable) UIUserNotificationSettings *currentUserNotificationSettings API_DEPRECATED("Use UserNotifications Framework's -[UNUserNotificationCenter getNotificationSettingsWithCompletionHandler:] and -[UNUserNotificationCenter getNotificationCategoriesWithCompletionHandler:]", ios(8.0, 10.0)) API_UNAVAILABLE(tvos);
                                                                         ^
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:253:70: note: 'currentUserNotificationSettings' has been explicitly marked deprecated here
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/NotificationPermissionStrategy.m:76:26: warning: 'UIUserNotificationTypeNone' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
        if (setting.types == UIUserNotificationTypeNone) permissionStatus = PermissionStatusDenied;
                             ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/permission_handler/permission_handler-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
    10 warnings generated.
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:50:9: warning: 'ABAuthorizationStatus' is deprecated: first deprecated in iOS 9.0 - use CNAuthorizationStatus [-Wdeprecated-declarations]
            ABAuthorizationStatus status = ABAddressBookGetAuthorizationStatus();
            ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:44:26: note: 'ABAuthorizationStatus' has been explicitly marked deprecated here
    typedef CF_ENUM(CFIndex, ABAuthorizationStatus) {
                             ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:50:40: warning: 'ABAddressBookGetAuthorizationStatus' is deprecated: first deprecated in iOS 9.0 - use [CNContactStore authorizationStatusForEntityType:] [-Wdeprecated-declarations]
            ABAuthorizationStatus status = ABAddressBookGetAuthorizationStatus();
                                           ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:50:33: note: 'ABAddressBookGetAuthorizationStatus' has been explicitly marked deprecated here
    AB_EXTERN ABAuthorizationStatus ABAddressBookGetAuthorizationStatus(void) AB_DEPRECATED("use [CNContactStore authorizationStatusForEntityType:]");
                                    ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:53:18: warning: 'kABAuthorizationStatusNotDetermined' is deprecated: first deprecated in iOS 9.0 - use CNAuthorizationStatus [-Wdeprecated-declarations]
                case kABAuthorizationStatusNotDetermined:
                     ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:44:26: note: 'ABAuthorizationStatus' has been explicitly marked deprecated here
    typedef CF_ENUM(CFIndex, ABAuthorizationStatus) {
                             ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:55:18: warning: 'kABAuthorizationStatusRestricted' is deprecated: first deprecated in iOS 9.0 - use CNAuthorizationStatus [-Wdeprecated-declarations]
                case kABAuthorizationStatusRestricted:
                     ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:44:26: note: 'ABAuthorizationStatus' has been explicitly marked deprecated here
    typedef CF_ENUM(CFIndex, ABAuthorizationStatus) {
                             ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:57:18: warning: 'kABAuthorizationStatusDenied' is deprecated: first deprecated in iOS 9.0 - use CNAuthorizationStatus [-Wdeprecated-declarations]
                case kABAuthorizationStatusDenied:
                     ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:44:26: note: 'ABAuthorizationStatus' has been explicitly marked deprecated here
    typedef CF_ENUM(CFIndex, ABAuthorizationStatus) {
                             ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:59:18: warning: 'kABAuthorizationStatusAuthorized' is deprecated: first deprecated in iOS 9.0 - use CNAuthorizationStatus [-Wdeprecated-declarations]
                case kABAuthorizationStatusAuthorized:
                     ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:44:26: note: 'ABAuthorizationStatus' has been explicitly marked deprecated here
    typedef CF_ENUM(CFIndex, ABAuthorizationStatus) {
                             ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:80:5: warning: 'ABAddressBookRequestAccessWithCompletion' is deprecated: first deprecated in iOS 9.0 - use [CNContactStore requestAccessForEntityType:completionHandler:] [-Wdeprecated-declarations]
        ABAddressBookRequestAccessWithCompletion(ABAddressBookCreate(), ^(bool granted, CFErrorRef error) {
        ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:73:16: note: 'ABAddressBookRequestAccessWithCompletion' has been explicitly marked deprecated here
    AB_EXTERN void ABAddressBookRequestAccessWithCompletion(ABAddressBookRef addressBook,  ABAddressBookRequestAccessCompletionHandler completion) AB_DEPRECATED("use [CNContactStore requestAccessForEntityType:completionHandler:]");
                   ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.m:80:46: warning: 'ABAddressBookCreate' is deprecated: first deprecated in iOS 9.0 - use [[CNContactStore alloc] init] [-Wdeprecated-declarations]
        ABAddressBookRequestAccessWithCompletion(ABAddressBookCreate(), ^(bool granted, CFErrorRef error) {
                                                 ^
    In module 'AddressBook' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/permission_handler-6.1.1/ios/Classes/strategies/ContactPermissionStrategy.h:11:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AddressBook.framework/Headers/ABAddressBook.h:60:28: note: 'ABAddressBookCreate' has been explicitly marked deprecated here
    AB_EXTERN ABAddressBookRef ABAddressBookCreate(void) AB_DEPRECATED("use [[CNContactStore alloc] init]");
                               ^
    8 warnings generated.
    1 warning generated.
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1461:38: warning: variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1426:3: note: variable 'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1461:48: warning: variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1425:3: note: variable 'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1550:38: warning: variable 'sum_left' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                         ^~~~~~~~
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1509:3: note: variable 'sum_left' is declared here
      uint16x8_t sum_left;
      ^
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1550:48: warning: variable 'sum_top' may be uninitialized when used here [-Wconditional-uninitialized]
        const uint16x8_t sum = vaddq_u16(sum_left, sum_top);
                                                   ^~~~~~~
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/dsp/dec_neon.c:1508:3: note: variable 'sum_top' is declared here
      uint16x8_t sum_top;
      ^
    4 warnings generated.
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/enc/backward_references_enc.c:694:12: warning: variable 'best_offset' may be uninitialized when used here [-Wconditional-uninitialized]
              (best_offset << MAX_LENGTH_BITS) | (uint32_t)best_length;
               ^~~~~~~~~~~
    /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/libwebp/src/enc/backward_references_enc.c:626:20: note: initialize the variable 'best_offset' to silence this warning
        int best_offset;
                       ^
                        = 0
    1 warning generated.
    1 warning generated.
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/image_picker-0.7.3/ios/Classes/FLTImagePickerPhotoAssetUtil.m:17:44: warning: 'UIImagePickerControllerReferenceURL' is deprecated: first deprecated in iOS 11.0 - Will be removed in a future release, use PHPicker. [-Wdeprecated-declarations]
      NSURL *referenceURL = [info objectForKey:UIImagePickerControllerReferenceURL];
                                               ^
    In module 'UIKit' imported from /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/ios/Pods/Target Support Files/image_picker/image_picker-prefix.pch:2:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIImagePickerController.h:62:51: note: 'UIImagePickerControllerReferenceURL' has been explicitly marked deprecated here
    UIKIT_EXTERN UIImagePickerControllerInfoKey const UIImagePickerControllerReferenceURL API_DEPRECATED("Will be removed in a future release, use PHPicker.", ios(4.1, 11.0)) API_UNAVAILABLE(tvos); // an NSURL that references an asset in the AssetsLibrary framework
                                                      ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/image_picker-0.7.3/ios/Classes/FLTImagePickerPhotoAssetUtil.m:21:47: warning: 'fetchAssetsWithALAssetURLs:options:' is deprecated: first deprecated in iOS 11 - Will be removed in a future release [-Wdeprecated-declarations]
      PHFetchResult<PHAsset *> *result = [PHAsset fetchAssetsWithALAssetURLs:@[ referenceURL ]
                                                  ^
    In module 'Photos' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/image_picker-0.7.3/ios/Classes/FLTImagePickerPhotoAssetUtil.h:6:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/Photos.framework/Headers/PHAsset.h:74:1: note: 'fetchAssetsWithALAssetURLs:options:' has been explicitly marked deprecated here
    + (PHFetchResult<PHAsset *> *)fetchAssetsWithALAssetURLs:(NSArray<NSURL *> *)assetURLs options:(nullable PHFetchOptions *)options API_DEPRECATED("Will be removed in a future release", ios(8, 11), tvos(8, 11)) API_UNAVAILABLE(macos);
    ^
    2 warnings generated.
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:55:1: warning: implementing deprecated method [-Wdeprecated-implementations]
    - (void)captureOutput:(AVCapturePhotoOutput *)output
    ^
    In module 'AVFoundation' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhotoOutput.h:753:1: note: method 'captureOutput:didFinishProcessingPhotoSampleBuffer:previewPhotoSampleBuffer:resolvedSettings:bracketSettings:error:' declared here
    - (void)captureOutput:(AVCapturePhotoOutput *)output didFinishProcessingPhotoSampleBuffer:(nullable CMSampleBufferRef)photoSampleBuffer previewPhotoSampleBuffer:(nullable CMSampleBufferRef)previewPhotoSampleBuffer resolvedSettings:(AVCaptureResolvedPhotoSettings *)resolvedSettings bracketSettings:(nullable AVCaptureBracketedStillImageSettings *)bracketSettings error:(nullable NSError *)error API_DEPRECATED_WITH_REPLACEMENT("-captureOutput:didFinishProcessingPhoto:error:", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:68:7: warning: 'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' is deprecated: first deprecated in iOS 11.0 [-Wdeprecated-declarations]
          JPEGPhotoDataRepresentationForJPEGSampleBuffer:photoSampleBuffer
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          -[AVCapturePhoto fileDataRepresentation]
    In module 'AVFoundation' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhotoOutput.h:544:1: note: 'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' has been explicitly marked deprecated here
    + (nullable NSData *)JPEGPhotoDataRepresentationForJPEGSampleBuffer:(CMSampleBufferRef)JPEGSampleBuffer previewPhotoSampleBuffer:(nullable CMSampleBufferRef)previewPhotoSampleBuffer API_DEPRECATED_WITH_REPLACEMENT("-[AVCapturePhoto fileDataRepresentation]", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:568:13: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
        while (!OSAtomicCompareAndSwapPtrBarrier(old, newBuffer, (void **)&_latestPixelBuffer)) {
                ^
    In module 'Darwin' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note: 'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:778:11: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
      while (!OSAtomicCompareAndSwapPtrBarrier(pixelBuffer, nil, (void **)&_latestPixelBuffer)) {
              ^
    In module 'Darwin' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note: 'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:1113:36: warning: 'AVVideoCodecH264' is deprecated: first deprecated in iOS 11.0 [-Wdeprecated-declarations]
          dictionaryWithObjectsAndKeys:AVVideoCodecH264, AVVideoCodecKey,
                                       ^~~~~~~~~~~~~~~~
                                       AVVideoCodecTypeH264
    In module 'AVFoundation' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/camera-0.8.0/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVVideoSettings.h:57:29: note: 'AVVideoCodecH264' has been explicitly marked deprecated here
            AVF_EXPORT NSString *const AVVideoCodecH264 /* @"avc1" */                               API_DEPRECATED_WITH_REPLACEMENT("AVVideoCodecTypeH264", macos(10.7, 10.13), ios(4.0, 11.0), tvos(9.0, 11.0)) API_UNAVAILABLE(watchos);
                                       ^
    5 warnings generated.
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataExif.m:9:9: warning: non-portable path to file '"SYMetadataExif.h"'; specified path differs in case from file name on disk [-Wnonportable-include-path]
    #import "SYMetadataEXIF.h"
            ^~~~~~~~~~~~~~~~~~
            "SYMetadataExif.h"
    /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataExif.m:41:65: warning: 'kCGImagePropertyExifSubsecTimeOrginal' is deprecated: first deprecated in iOS 10.0 - No longer supported [-Wdeprecated-declarations]
                 SYStringSel(subsecTimeOriginal):       (NSString *)kCGImagePropertyExifSubsecTimeOrginal,
                                                                    ^
    In module 'ImageIO' imported from /Users/runner/hostedtoolcache/flutter/2.0.2-stable/x64/.pub-cache/hosted/pub.dartlang.org/flutter_image_compress-0.7.0/ios/Classes/SYPictureMetadata/SYMetadataExif.m:10:
    /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk/System/Library/Frameworks/ImageIO.framework/Headers/CGImageProperties.h:248:34: note: 'kCGImagePropertyExifSubsecTimeOrginal' has been explicitly marked deprecated here
    IMAGEIO_EXTERN const CFStringRef kCGImagePropertyExifSubsecTimeOrginal IMAGEIO_AVAILABLE_BUT_DEPRECATED(10.4, 10.11, 4.0, 10.0);
                                     ^
    2 warnings generated.
    12 warnings generated.
    ld: warning: ignoring file /Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/build/ios/Release-iphoneos/libwebp/libwebp.framework/libwebp, building for iOS-armv7 but attempting to link with file built for iOS-arm64
    Undefined symbols for architecture armv7:
      "_WebPMemoryWriterInit", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPPictureImportRGB", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPConfigInitInternal", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPPictureRescale", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPEncode", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPMemoryWriterClear", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPPictureFree", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPNewInternal", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPMuxSetAnimationParams", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPMuxDelete", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPMuxAssemble", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPFree", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPPictureImportRGBA", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPDemuxGetChunk", referenced from:
          -[SDImageWebPCoder sd_createColorSpaceWithDemuxer:] in SDImageWebPCoder.o
      "_WebPDemuxReleaseChunkIterator", referenced from:
          -[SDImageWebPCoder sd_createColorSpaceWithDemuxer:] in SDImageWebPCoder.o
      "_WebPInitDecoderConfigInternal", referenced from:
          -[SDImageWebPCoder sd_createWebpImageWithData:colorSpace:scaledSize:] in SDImageWebPCoder.o
      "_WebPDemuxReleaseIterator", referenced from:
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder scanAndCheckFramesValidWithDemuxer:] in SDImageWebPCoder.o
          -[SDImageWebPCoder safeStaticImageFrame] in SDImageWebPCoder.o
          -[SDImageWebPCoder safeAnimatedImageFrameAtIndex:] in SDImageWebPCoder.o
      "_WebPIDecGetRGB", referenced from:
          -[SDImageWebPCoder incrementalDecodedImageWithOptions:] in SDImageWebPCoder.o
      "_WebPDemuxGetFrame", referenced from:
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder scanAndCheckFramesValidWithDemuxer:] in SDImageWebPCoder.o
          -[SDImageWebPCoder safeStaticImageFrame] in SDImageWebPCoder.o
          -[SDImageWebPCoder safeAnimatedImageFrameAtIndex:] in SDImageWebPCoder.o
      "_WebPMemoryWrite", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPDecode", referenced from:
          -[SDImageWebPCoder sd_createWebpImageWithData:colorSpace:scaledSize:] in SDImageWebPCoder.o
      "_WebPIUpdate", referenced from:
          -[SDImageWebPCoder updateIncrementalData:finished:] in SDImageWebPCoder.o
      "_WebPINewRGB", referenced from:
          -[SDImageWebPCoder initIncrementalWithOptions:] in SDImageWebPCoder.o
      "_WebPIDelete", referenced from:
          -[SDImageWebPCoder dealloc] in SDImageWebPCoder.o
      "_WebPGetFeaturesInternal", referenced from:
          -[SDImageWebPCoder sd_createWebpImageWithData:colorSpace:scaledSize:] in SDImageWebPCoder.o
      "_WebPDemuxGetI", referenced from:
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder sd_createColorSpaceWithDemuxer:] in SDImageWebPCoder.o
          -[SDImageWebPCoder scanAndCheckFramesValidWithDemuxer:] in SDImageWebPCoder.o
      "_WebPDemuxInternal", referenced from:
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder updateIncrementalData:finished:] in SDImageWebPCoder.o
          -[SDImageWebPCoder initWithAnimatedImageData:options:] in SDImageWebPCoder.o
      "_WebPDemuxNextFrame", referenced from:
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder scanAndCheckFramesValidWithDemuxer:] in SDImageWebPCoder.o
          -[SDImageWebPCoder safeAnimatedImageFrameAtIndex:] in SDImageWebPCoder.o
      "_WebPMuxPushFrame", referenced from:
          -[SDImageWebPCoder encodedDataWithImage:format:options:] in SDImageWebPCoder.o
      "_WebPPictureInitInternal", referenced from:
          -[SDImageWebPCoder sd_encodedWebpDataWithImage:quality:maxPixelSize:maxFileSize:options:] in SDImageWebPCoder.o
      "_WebPDemuxDelete", referenced from:
          -[SDImageWebPCoder dealloc] in SDImageWebPCoder.o
          -[SDImageWebPCoder decodedImageWithData:options:] in SDImageWebPCoder.o
          -[SDImageWebPCoder updateIncrementalData:finished:] in SDImageWebPCoder.o
          -[SDImageWebPCoder initWithAnimatedImageData:options:] in SDImageWebPCoder.o
    ld: symbol(s) not found for architecture armv7
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ld: warning: dylib (/Users/runner/work/hanglyde_mobile_prod/hanglyde_mobile_prod/build/ios/Release-iphoneos/libwebp/libwebp.framework/libwebp) was built for newer iOS version (12.0) than being linked (9.0)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Encountered error while building for device.
Error: Process completed with exit code 1.
-->

<details>
  <pre> In Xcode the build archives with no issues and bundle exec fastlane release archives the build with no problems. When running in github actions though it spits out that error (  ld: symbol(s) not found for architecture armv7). No matter what I configure in Xcode I still cannot get this error to go away.  </pre>
</details>

### Environment

<!-- 
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 11.2.3                                                         |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | true                                                           |
| Git                         | git version 2.29.2                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 11.2.3 (20D91)                                           |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 12.4                                                           |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane


DEVELOPER_APP_ID = ENV["DEVELOPER_APP_ID"]
DEVELOPER_APP_IDENTIFIER = ENV["DEVELOPER_APP_IDENTIFIER"]
PROVISIONING_PROFILE_SPECIFIER = ENV["PROVISIONING_PROFILE_SPECIFIER"]
TEMP_KEYCHAIN_USER = ENV["TEMP_KEYCHAIN_USER"]
TEMP_KEYCHAIN_PASSWORD = ENV["TEMP_KEYCHAIN_PASSWORD"]

default_platform(:ios)
platform:ios do
    if is_ci
        create_keychain(
          name: ENV['MATCH_KEYCHAIN_NAME'],
          password: ENV["MATCH_KEYCHAIN_PASSWORD"],
          default_keychain: true,
          unlock: true,
          timeout: 1800,
          lock_when_sleeps: false
        )
      end

    match(
      type: 'appstore',
      app_identifier: "#{DEVELOPER_APP_IDENTIFIER}",
      #git_basic_authorization: Base64.strict_encode64(ENV["GIT_AUTHORIZATION"]),
      ##readonly: true,
      readonly: is_ci,
      keychain_name: ENV['MATCH_KEYCHAIN_NAME'],
      keychain_password: ENV["MATCH_KEYCHAIN_PASSWORD"],
    )

    desc "Push a new build to Production Release"
    lane :release do
        sync_code_signing(type: "appstore")
        unlock_keychain 
        build_app(scheme: "Runner") #change name of workspace
        set_release_version
        upload_to_app_store(submission_information: {
          export_compliance_platform: 'ios',
          export_compliance_uses_encryption: false,
      },
          username: "devteam@hanglyde.com", app_version: get_version_number(
            xcodeproj: "Runner.xcodeproj",
            target: "Runner"
          ),submit_for_review: true)
    end

    desc "Sets the version of the bundle to a RELEASE_VERSION passed in as an environment variable"
    lane:set_release_version do
        version = get_version_number(
            xcodeproj: "Runner.xcodeproj",
            target: "Runner"
          )
        if version
            UI.message("Setting version to #{version}")
            increment_version_number(bump_type: "patch")
            increment_build_number(build_number: Time.now.to_i)
        else
            UI.user_error!("Environment variable RELEASE_VERSION not set")
        end
    end
end
# lane :release do             # generate new screenshots for the App Store
#   sync_code_signing(type: "appstore")  # see code signing guide for more information
#   build_app(scheme: "Runner")
#   upload_to_app_store                  # upload your app to App Store Connect
#   slack(message: "Successfully uploaded a new App Store build")
# end

# def delete_temp_keychain(name)
#   delete_keychain(
#     name: name
#   ) if File.exist? File.expand_path("~/Library/Keychains/#{name}-db")
# end

# def create_temp_keychain(name, password)
#   create_keychain(
#     name: name,
#     password: password,
#     unlock: false,
#     timeout: false
#   )
# end

# def ensure_temp_keychain(name, password)
#   delete_temp_keychain(name)
#   create_temp_keychain(name, password)
# end

# platform :ios do
#   lane :release do
#     match(
#       type: 'appstore',
#       app_identifier: "#{DEVELOPER_APP_IDENTIFIER}",
#       git_basic_authorization: Base64.strict_encode64(ENV["GIT_AUTHORIZATION"]),
#       readonly: true,
#     )

#     gym(
#       configuration: "Release",
#       workspace: "Runner.xcworkspace",
#       scheme: "Runner",
#       export_method: "app-store",
#       export_options: {
#         provisioningProfiles: { 
#             DEVELOPER_APP_ID => PROVISIONING_PROFILE_SPECIFIER
#         }
#       }
#     )
#    sync_code_signing(type: "appstore")  # see code signing guide for more information
#    build_app(scheme: "Runner")
#    upload_to_app_store                  # upload your app to App Store Connect

#   end
# end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
#app_identifier("com.hanglyde.Hanglyde") # The bundle identifier of your app
#apple_id("devteam@hanglyde.com") # Your Apple email address

#itc_team_id("120229567") # App Store Connect Team ID
#team_id("BMJZ47A2SL") # Developer Portal Team ID
app_identifier(ENV["DEVELOPER_APP_IDENTIFIER"])
apple_id(ENV["FASTLANE_APPLE_ID"])

itc_team_id(ENV["APP_STORE_CONNECT_TEAM_ID"])
team_id(ENV["DEVELOPER_PORTAL_TEAM_ID"])

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.179.0 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

| Plugin                          | Version | Update-Status |
| ------------------------------- | ------- | ------------- |
| fastlane-plugin-flutter_version | 1.0.1   | ✅ Up-To-Date  |


<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.4.0        |
| bundler                         | 2.2.15       |
| rake                            | 13.0.3       |
| CFPropertyList                  | 3.0.3        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.7.0        |
| artifactory                     | 3.0.15       |
| atomos                          | 0.1.3        |
| aws-eventstream                 | 1.1.1        |
| aws-partitions                  | 1.435.0      |
| aws-sigv4                       | 1.2.3        |
| jmespath                        | 1.4.0        |
| aws-sdk-core                    | 3.113.0      |
| aws-sdk-kms                     | 1.43.0       |
| aws-sdk-s3                      | 1.93.0       |
| babosa                          | 1.0.4        |
| claide                          | 1.0.3        |
| colored                         | 1.2          |
| colored2                        | 3.1.2        |
| highline                        | 1.7.10       |
| commander-fastlane              | 4.4.6        |
| declarative                     | 0.0.20       |
| declarative-option              | 0.1.0        |
| digest-crc                      | 0.6.3        |
| unf_ext                         | 0.0.7.7      |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| dotenv                          | 2.7.6        |
| emoji_regex                     | 3.2.2        |
| excon                           | 0.79.0       |
| faraday-net_http                | 1.0.1        |
| multipart-post                  | 2.0.0        |
| ruby2_keywords                  | 0.0.4        |
| faraday                         | 1.3.0        |
| http-cookie                     | 1.0.3        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.0.0        |
| fastimage                       | 2.2.3        |
| gh_inspector                    | 1.1.3        |
| jwt                             | 2.2.2        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.1        |
| signet                          | 0.15.0       |
| googleauth                      | 0.16.0       |
| httpclient                      | 2.8.3        |
| mini_mime                       | 1.0.3        |
| uber                            | 0.1.0        |
| representable                   | 3.0.4        |
| retriable                       | 3.1.2        |
| google-api-client               | 0.38.0       |
| rexml                           | 3.2.4        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.3.0        |
| google-apis-iamcredentials_v1   | 0.2.0        |
| google-apis-storage_v1          | 0.3.0        |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.1.0        |
| google-cloud-core               | 1.6.0        |
| google-cloud-storage            | 1.31.0       |
| json                            | 2.5.1        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| plist                           | 3.6.0        |
| rubyzip                         | 2.3.0        |
| security                        | 0.1.3        |
| simctl                          | 1.6.8        |
| slack-notifier                  | 2.3.2        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.7.0        |
| terminal-table                  | 1.8.0        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.19.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
| fastlane-plugin-flutter_version | 1.0.1        |
</details>


*generated on:* **2021-03-30**
</details>
-->

<details>
  <pre> Github Action: 
# This is a basic workflow to help you get started with Actions
name: CI
# Controls when the action will run. 
on:
  # Triggers the workflow on push or pull request events but only for the master branch
  pull_request:
    branches: [ master ]
  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
 deploy_ios:
    name: Deploy Production build to AppStore
    runs-on: macOS-latest
    steps:
      - name: Checkout code from ref
        uses: actions/checkout@v1
        with:
          ref: ${{ github.ref }}
      - name: Run Flutter tasks
        uses: subosito/flutter-action@v1
        with:
          flutter-version: '2.0.2'
      - run: flutter pub get
      - run: flutter build ios --release --no-codesign
      - name: Deploy iOS Production App to AppStore via Fastlane
        uses: maierj/fastlane-action@v2.0.1
        with:
          lane: release
          subdirectory: ios
        env:
          APP_STORE_CONNECT_TEAM_ID: '${{ secrets.APP_STORE_CONNECT_TEAM_ID }}'
          DEVELOPER_APP_ID: '${{ secrets.DEVELOPER_APP_ID }}'
          DEVELOPER_APP_IDENTIFIER: '${{ secrets.DEVELOPER_APP_IDENTIFIER }}'
          DEVELOPER_PORTAL_TEAM_ID: '${{ secrets.DEVELOPER_PORTAL_TEAM_ID }}'
          FASTLANE_APPLE_ID: '${{ secrets.FASTLANE_APPLE_ID }}'
          FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: '${{ secrets.FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD }}'
          MATCH_PASSWORD: '${{ secrets.MATCH_PASSWORD }}'
          GIT_AUTHORIZATION: '${{ secrets.GIT_AUTHORIZATION }}'
          PROVISIONING_PROFILE_SPECIFIER: '${{ secrets.PROVISIONING_PROFILE_SPECIFIER }}'
          TEMP_KEYCHAIN_PASSWORD: '${{ secrets.TEMP_KEYCHAIN_PASSWORD }}'
          TEMP_KEYCHAIN_USER: '${{ secrets.TEMP_KEYCHAIN_USER }}'
 </pre>
</details>

**Comment by Hwarne1995** [comment]
You have not linked against the correct libz file. If you right click the file and reveal in finder its path should be somewhere in an iOS sdk folder. Here is mine for example

`/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS4.3.sdk/usr/lib`

I recommend removing the reference and then re-adding it back in the Link Binary With Libraries section Build Phases of your target.

**Comment by Amanda-45** [comment]
Xcode's output: ↳ Undefined symbols for architecture armv7: Code · Issues 5k+ · Pull requests 178 · Actions · Projects 139 · Wiki · Security · Insights  in FlutterWebView.o ld: symbol(s) not found for architecture armv7 clang: error: linker command failed with exit  How are you trying to archive the build?

**Comment by isis-hanglyde** [comment]
@Hwarne1995 I tried that with no luck! I also upgraded to the latest version of fastlane.
